game_board.py

```diff
@@ -39,20 +47,45 @@ def map_cord(str_board):
             map_xy.append(char_xy)
             x += 1
         y += 1
+    
     return map_xy
 
-def move_plus():
+def make_plus_list():
     """
     Finds all the "+" chars in the map_xy 
     Returns a list of lists where every list has ["+",xPos,Ypos]
     Example [['X', 0, 0], [' ', 1, 0],...,['+', '4', '0']] => [['+', '4', '0'],...]
     """
-    map_xy =map_cord(read_map(map_path))
-    move = []
+    map_xy =convert_map_to_coordinates(read_map(map_path))
+    plus_list = []
     for cord in map_xy:
         if cord[0] == "+":
-            move.append([cord[0],str(cord[1]),str(cord[2])])
-    return move
+            plus_list.append([cord[0],str(cord[1]),str(cord[2])])
+    return plus_list
+
+def print_player_names(screen,player1_name,player2_name):
+    """Prints the playernames in screen 
+
+    Keyword arguments:
+    screen -- the curses screen
+    player1_name -- player1 name as string
+    player2_name -- player1 name as string
+    """
+    curses.init_pair(3,curses.COLOR_RED,curses.COLOR_BLACK)
+    h,w = screen.getmaxyx()
+
+    ply1_x = -1 + w//3   
+    ply1_y = 2
```

Comment:
Don't be afraid to write long variable names. It's better to write longer variable names that clearly describe what they represent than using shorter names that aren't as descriptive.
I would suggest renaming them to `player1_name_x` and `player1_name_y`